configure.ac

```diff
@@ -31,12 +31,12 @@ PKG_CHECK_MODULES(CAIRO, cairo >= $CAIRO_REQUIRED_VERSION)
 # Optional use of Pango's text rendering on top of Cairo
 AC_ARG_WITH(pango, [  --with-pango            use Pango to measure and draw text],[text_v=pango],[text_v=default])
 
-GDIPLUS_CFLAGS="$GDIPLUS_CFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -std=gnu99"
+GDIPLUS_CFLAGS="$GDIPLUS_CFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -std=gnu11"
```

Comment:
I have a hunch we might not have a C11 compiler everywhere ...

Can you please try whether something like this fixes the warning too?

```c
#ifndef UINT16
typedef guint16 UINT16;
#endif
```

Reply:
Does not work, sadly

Reply:
> I have a hunch we might not have a C11 compiler everywhere ...

Are we not just then lying because the code shouldn't currently comply with a compiler that only supports c99!

Reply:
yeah but e.g. CentOS 6 ships with an ancient GCC which doesn't support C11 and it was not an issue there.

I just learned though that we build with a newer GCC on that old distro anyway so this is probably fine.